docs/view_components/welcome.rst

```diff
@@ -2,17 +2,17 @@ linking ************
 Welcome page
 ************
 
-A welcome window can be seen in almost every software/website in production. The welcome window should serve as an orientation point for the user and let them know, in as brief words (and other audiovisual content) as possible, what to expect from the tool. It is also the launch point from where the user begins their energy systems simulation.
+The welcome window serves as an orientation point for the user and let them know, in as brief words (and other audiovisual content) as possible, what to expect from the tool. It is also the launch point from where the user begins their energy systems simulation.
 
 The welcome page is to be implemented as an optional pop-up that springs up on the landing page when the user launches the tool. In this section, a structure for the welcome page is proposed, subject to further discussion with the stakeholders.
 
 Attributes
 ----------
 
-The following are the view-components available to the user in the welcome pop-up on the landing window immediately after launching the software for the first time:
+The following are the components available to the user in the welcome pop-up on the landing window immediately after launching the software for the first time:
```

Comment:
We actually don't need an introductory text for what attributes are, this belongs to the concept definition